api/setup.php

```diff
@@ -1,16 +1,16 @@
 <?php
 /**
- * Route Newspack Campaigns API requests based on method.
+ * Setup Campaigns API.
  *
  * @package Newspack
  */
 
-$plugin_path = substr($_SERVER['SCRIPT_FILENAME'], 0, strrpos($_SERVER['SCRIPT_FILENAME'], 'wp-content/plugins/')); // phpcs:ignore
+$wp_root_path = substr($_SERVER['SCRIPT_FILENAME'], 0, strrpos($_SERVER['SCRIPT_FILENAME'], 'wp-content/plugins/')); // phpcs:ignore
```

Comment:
This issue is a holdover from past PRs but we should go more specific with the PHPCS ignore rules. Using blanket ignore rules makes it easy to miss basic formatting issues, like spaces after parentheses in this line.

Reply:
Done in e5a08a65cce36564e863cd12cc05c85404139681